zipkin-lens/src/components/App/Layout.test.js

```diff
@@ -1,62 +0,0 @@
-/*
```

Comment:
I thought these unit tests were meaningless and would prevent us from changing structures of the DOM tree in the future.
So I deleted these unit tests.

Reply:
Was gonna ask about this, thanks for explaining your reasoning.